app/src/main/java/com/practicum/playlistmaker/favorite/domain/db/FavoriteInteractor.kt

```diff
@@ -0,0 +1,11 @@
+package com.practicum.playlistmaker.favorite.domain.db
```

Comment:
🍏  Так как в этом пакете лежат интерфейсы, которые являются контрактами, можно назвать пакет api
```suggestion
package com.practicum.playlistmaker.favorite.domain.api
```